hack/common.sh

```diff
@@ -103,10 +103,23 @@ sti::build::build_binaries() {
     for platform in "${platforms[@]}"; do
       sti::build::set_platform_envs "${platform}"
       echo "++ Building go targets for ${platform}:" "${targets[@]}"
-      go install "${goflags[@]:+${goflags[@]}}" \
-          -ldflags "${version_ldflags}" \
-          "${binaries[@]}"
-      sti::build::unset_platform_envs "${platform}"
+      # have to enable these debug flags for using gdb separately from goflags
+      # because the -gcflags "-N -l" ... i.e. the -N and -l are the setting for -gcflags
+      # not for go itself .. for now we punt on other goflags if developers are
+      # compiling to run with gdb
+      if [[ -v STI_ENABLE_GDB ]]; then
+	  echo "Enabling compiling go with debug to allow gdb usage (i.e. -gcflags \"-N -l\") since STI_ENABLE_GDB is set"
+	  echo "In this case, STI_ENABLE_GDB is set to  ${STI_ENABLE_GDB}"
```

Comment:
I am assuming you mean the 2nd echo ... I can remove that, and just leave the first echo that confirms that since the env is in fact set, we are employing -gcflags ...